src/ui/common/src/reducers/workflow.ts

```diff
@@ -156,6 +193,26 @@ export const workflowSlice = createSlice({
     },
   },
   extraReducers: (builder) => {
+    builder.addCase(handleGetSelectDagPosition.pending, (state, action) => {
```

Comment:
Where are we storing the selectedDagPosition so that we can avoid having to refetch?